matminer/data_retrieval/retrieve_Citrine.py

```diff
@@ -27,150 +29,146 @@ def get_value(dict_item):
+                                p_df[prop["name"]] = prop["vectors"]
+                            elif "matrices" in prop:
+                                p_df[prop["name"]] = prop["matrices"]
+                            # parse all keys in the Property object except 'name', 'scalars', 'vectors', and 'matrices'
+                            for prop_key in prop:
+                                if prop_key not in ["name", "scalars", "vectors", "matrices"]:
+                                    # If value is a list of multiple items, set the cell to the entire list by first
+                                    if isinstance(prop[prop_key], list) and len(prop[prop_key])>1:
+                                        p_df[prop["name"] + "-" + prop_key] = np.nan
+                                        p_df[prop["name"] + "-" + prop_key] = \
+                                            p_df[prop["name"] + "-" + prop_key].astype(object)
+                                    p_df.set_value(counter, prop["name"] + "-" + prop_key, prop[prop_key])
+                        p_df.index = [counter]
+                        prop_df = prop_df.append(p_df)
+            df_prop = pd.concat([non_prop_df, prop_df], axis=1)
+            if prop_counter==0:
+                optcomcols = df_prop.columns.values
+            else:
+                optcomcols = list(set(optcomcols) & set(df_prop.columns))
+            all_fields += list(df_prop.columns.values)
+            if not secondary_fields:
+                outcols = []
+                for p in df_prop.columns.values:
+                    if not requested_prop or requested_prop in p:
+                        outcols.append(p)
+                df_prop = df_prop[outcols + ["uid"] + common_fields]
+            if prop_counter == 0:
+                df = df_prop
+            else:
+                if "uid" in df and "uid" in df_prop:
+                    uid = ["uid"]
+                else:
+                    uid = []
+                try:
+                    if len(uid+common_fields) > 0:
+                        df = df.merge(df_prop, on=uid+common_fields, how="outer")
+                    else:
+                        df = df.join(df_prop, how="outer")
+                except TypeError or KeyError:
+                    raise TypeError('Use scalar/string fields for common_fields'
+                                    'common_fields among: {}'.format(optcomcols))
+        uninformative_columns = ["category", "uid"]
+        optcomcols = [c for c in optcomcols if c not in uninformative_columns]
+        for col in uninformative_columns:
+            if col in df:
+                df = df.drop(col, axis=1)
+        if print_properties_options:
+            print("all available fields:\n{}".format(list(set(all_fields))))
+            print("\nsuggested common fields:\n{}".format(optcomcols))
         return df
 
     def get_api_data(self, formula=None, prop=None, data_type=None,
```

Comment:
do you want to just call this ``get_data`` instead of get_api_data?

Reply:
Yes, this is the perfect usage for get_data. Its since been changed